action.yml

```diff
@@ -32,13 +32,11 @@ inputs:
     default: 'false'
 
 runs:
-  using: composite
-  steps:
-    - shell: bash
-      run: $GITHUB_ACTION_PATH/entrypoint.sh
-      env:
-        INPUT_IMAGE: ${{ inputs.image }}
-        INPUT_SOURCE_TAG: ${{ inputs.source_tag }}
-        INPUT_DEST_TAG: ${{ inputs.dest_tag }}
-        INPUT_VERBOSE: ${{ inputs.verbose }}
-        INPUT_DRY: ${{ inputs.dry }}
+  using: docker
+  image: Dockerfile
```

Comment:
so is it going to build the docker image every time this runs?

Reply:
It does - it takes ~17s. I believe in the future if we want to we can pre-build this image and ship it as an artifact. Given that this only runs for a release-action, I am not worried about it at the moment.